components/resourceDetails/resourceSharing/agentAccessList/__snapshots__/index.test.jsx.snap

```diff
@@ -35,24 +35,8 @@ exports[`AgentAccessList renders an empty list of permissions 1`] = `
 
 exports[`AgentAccessList renders loading while loading permissions for access control 1`] = `
 <DocumentFragment>
-  <div
-    class="PodBrowser-root PodBrowser-colorPrimary PodBrowser-indeterminate"
-    role="progressbar"
-    style="width: 40px; height: 40px;"
-  >
-    <svg
-      class="PodBrowser-svg"
-      viewBox="22 22 44 44"
-    >
-      <circle
-        class="PodBrowser-circle PodBrowser-circleIndeterminate"
-        cx="44"
-        cy="44"
-        fill="none"
-        r="20.2"
-        stroke-width="3.6"
-      />
-    </svg>
+  <div>
```

Comment:
This snapshot should be rendering a loading spinner. The spinner does appear in the vercel deployment so it would appear it's something to do with the test itself.